lib/matplotlib/figure.py

```diff
@@ -1241,6 +1242,60 @@ def align_xlabels(self, axs=None):
                         # grouper for groups of xlabels to align
                         self._align_label_groups['x'].join(ax, axc)
 
+    def align_titles(self, axs=None):
```

Comment:
I wonder if we want to add `align_titles` to `align_labels`?    Though this would be a breaking change of `align_labels`.